# Define a custom message for target color information
Header header
string[] color_names     # Names of the target colors
float32[] proportions   # Relative proportions corresponding to the colors
float32[] r_values       # Red channel values for the colors
float32[] g_values       # Green channel values for the colors
float32[] b_values       # Blue channel values for the colors
